### PR TITLE
Add `myaws ec2ri ls` command

### PR DESCRIPTION
**Pull request by akiray03**
Lists EC2 Reserved Instances:
```
$ myaws ec2ri ls
2c08d8eb-b7a6-42fd-93b6-xxxxxxxxxxxx    active  Region  N/A     c4.large          2      1year  2016-10-06      2017-09-16
b98256d5-7f57-41f3-882d-xxxxxxxxxxxx    active  Region  N/A     m3.medium         2      1year  2016-10-13      2017-09-16
c6543670-6424-4546-891e-xxxxxxxxxxxx    active  Region  N/A     c4.2xlarge        4      1year  2016-10-13      2017-09-16
c9ba93a3-5acd-40e7-a0a2-xxxxxxxxxxxx    active  Region  N/A     c4.large          2      1year  2016-10-06      2017-09-16
e6758f15-638f-4be7-b64c-xxxxxxxxxxxx    active  Region  N/A     t2.medium         4      1year  2016-10-13      2017-09-16
```

### CODE REVIEW

File: myaws/ec2ri_ls.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
+package myaws
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"strings"
```
Comment (minamijoyo) [inline myaws/ec2ri_ls.go]: Please sort import packages with `goimports` rule. Standard libraries should be on top, third party's be later.

File: myaws/ec2ri_ls.go [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+)
+
+type EC2RILsOptions struct {
```
Comment (minamijoyo) [inline myaws/ec2ri_ls.go]: Add comment to the public type.

File: myaws/ec2ri_ls.go [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+	All    bool
+	Quiet  bool
+	Fields []string
+}
+
+func (client *Client) EC2RILs(options EC2RILsOptions) error {
```
Comment (minamijoyo) [inline myaws/ec2ri_ls.go]: Add comment to the public method.

**Comment by akiray03** [comment]
@minamijoyo fixed it.

**Comment by minamijoyo** [comment]
LGTM 👍  Thanks !!

**Pull request merged**